nginx.conf

```diff
@@ -76,7 +76,6 @@ http {
 	##
 
 	include /etc/nginx/conf.d/*.conf;
-	include /etc/nginx/sites-enabled/*;
```

Comment:
The `sites-enabled` and `sites-available` used to be used in nginx for config. They no longer are, since a few versions ago. Your installation probably uses one of the older versions. All that this line is doing is say include conf files from `sites-enabled`. It's not used for anything other than loading available configs, which is now done in `conf.d` by default so it's sort of redundant to use both. Could use any directory you want realy, and just include them. This just updates the config to use the common practice for newer versions of nginx which the docker image is using. Should be backwards compatible since its just a matter of where the config file gets loaded from.